integration_tests/suite/test_calls_bus_consume.py

```diff
@@ -104,7 +107,152 @@ def assert_function():
                                 'status': 'Up',
                                 'hangup_time': None,
                                 'answer_time': is_(a_timestamp()),
-                                'is_video': True
+                                'is_video': True,
+                                'direction': 'outbound',
+                            })
+                        }),
+                        headers=has_entries(
+                            name='call_updated',
+                            tenant_uuid=VALID_TENANT,
+                        )
+                    )
+                )
+            )
+
+        until.assert_(assert_function, tries=5)
+
+    def test_when_channel_joins_bridge_call_direction_updated(self):
+        first_channel_id = new_call_id()
+        second_channel_id = new_call_id()
+        self.ari.set_channels(
+            MockChannel(
+                id=first_channel_id,
+                state='Up',
+                channelvars={
+                    'CHANNEL(videonativeformat)': '(vp8)',
+                    'WAZO_ANSWER_TIME': '2022-03-08T03:49:00+00:00',
+                },
+            ),
+            MockChannel(
+                id=second_channel_id,
+                state='Up',
+                channelvars={
+                    'CHANNEL(videonativeformat)': '(vp8)',
+                    'WAZO_ANSWER_TIME': '2022-03-08T03:49:10+00:00',
+                },
+            )
+        )
+        self.ari.set_channel_variable({
+            first_channel_id: {
+                'WAZO_TENANT_UUID': VALID_TENANT,
+            },
+            second_channel_id: {
+                'WAZO_TENANT_UUID': VALID_TENANT,
+                'WAZO_CALL_DIRECTION': 'outbound',
+            },
+        })
+
+        events = self.bus.accumulator(routing_key='calls.call.updated')
+
+        self.bus.send_ami_newstate_event(first_channel_id)
+        self.bus.send_ami_newchannel_event(second_channel_id)
+
+        def assert_function():
+            assert_that(
+                events.accumulate(with_headers=True),
+                has_item(
+                    has_entries(
+                        message=has_entries({
+                            'name': 'call_updated',
+                            'origin_uuid': XIVO_UUID,
+                            'data': has_entries({
+                                'call_id': first_channel_id,
+                                'status': 'Up',
+                                'hangup_time': None,
+                                'answer_time': is_(a_timestamp()),
+                                'is_video': True,
```

Comment:
We don't care about `answer_time` or `is_video` in this test. Please reduce the test to the minimum :)